mmit/core/piecewise_function.cpp

```diff
@@ -146,7 +146,7 @@ bool has_slack_at_position(double func_breakpoint_pos, double func_sign, Coeffic
     }
 }
 
-int PiecewiseFunction::insert_point(double y, bool is_upper_bound) {
+int PiecewiseFunction::insert_point(double b, Coefficients F, bool is_upper_bound) {
```

Comment:
if the is_upper_bound flag only used for multiplying the coefficients by the sign, you could do that before calling insert_point